httpcore5/src/main/java/org/apache/hc/core5/util/ReflectionUtils.java

```diff
@@ -44,12 +46,13 @@ public static void callSetter(final Object object, final String setterName, fina
         }
     }
 
-    public static <T> T callGetter(final Object object, final String getterName, final Class<T> resultType) {
+    public static <T> T callGetter(final Object object, final String getterName, final Object arg, final Class argType, final Class<T> resultType) {
```

Comment:
@kkewwei Is this method actually being used anywhere besides tests? Do we need to change it?

Reply:
It seems unreasonable to change the publish method. otheres may be depend on it.

Should I create the similar method?

Reply:
It indeed seem unreasonable to change the publish method. others may be depend on it.